catalog/task/skopeo-copy/0.2/README.md

```diff
@@ -0,0 +1,13 @@
+# push-application-snapshot
```

Comment:
All that changed in this task was the description of one of the parameters. I don't think that constitutes making a 0.2 version